tests/bluetooth/common/testlib/include/testlib/addr.h

```diff
@@ -17,4 +17,18 @@
 		.a = {{last, 0x00, 0x00, 0x00, 0x00, 0xc0}},                                       \
 	})
 
+static inline char *bt_testlib_print_address_(const bt_addr_le_t *addr,
+					      char addr_str[BT_ADDR_LE_STR_LEN])
+{
+	bt_addr_le_to_str(addr, addr_str, BT_ADDR_LE_STR_LEN);
+
+	return addr_str;
+}
```

Comment:
`print_address` is a bit misleading as it doesn't print it :)

Reply:
It's interesting, because it's effectively does the same as `bt_testlib_addr_to_str` just with 1 more parameter. Perhaps the names should reflect that? 



> Fair, what about `get_addr_`?

What about `get_addr_str_`? 

Do we really need the `_` suffix? It can be used on it's own, and probably should be treated as such.

Reply:
I don't see why we would use it on its own over just `bt_addr_le_to_str`. If we're already declaring multiple arrays to store the strings in we might just use that.

Reply:
> I don't see why we would use it on its own

Wasn't as much a question of whether we should, but users certainly _can_. 

Anyways, not a blocker :) Is the `_` suffix for private function a naming convention we use in Zephyr?

Reply:
True, I guess it's not used, removed it!